avocado/plugins/yaml_to_mux.py

```diff
@@ -398,6 +398,7 @@ def __iter__(self):
                 data.set_environment_dirty()
             yield {"variant_id": i,
                    "variant": data.get_leaves(),
+                   "variant_id_short": i,
```

Comment:
OK, although it looks like this and fd894f68bb84be77eb7d66d167b419efb0eb499b are so similar they could be one, but no hard opinions on that.